tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -403,28 +410,28 @@ const DeviceCommand EnqueueReadBufferCommand::assemble_device_command(uint32_t d
 }
 
 void EnqueueReadBufferCommand::process() {
-    uint32_t write_ptr = this->manager.get_issue_queue_write_ptr(this->command_queue_channel);
-    this->read_buffer_addr = this->manager.get_completion_queue_read_ptr(this->command_queue_channel);
+    uint32_t write_ptr = this->manager.get_issue_queue_write_ptr(this->id);
+    this->read_buffer_addr = this->manager.get_completion_queue_read_ptr(this->id);
 
     const auto cmd = this->assemble_device_command(this->read_buffer_addr);
 
-    this->manager.issue_queue_reserve_back(DeviceCommand::NUM_BYTES_IN_DEVICE_COMMAND, this->command_queue_channel);
+    this->manager.issue_queue_reserve_back(DeviceCommand::NUM_BYTES_IN_DEVICE_COMMAND, this->id);
     this->manager.cq_write(cmd.get_desc().data(), DeviceCommand::NUM_BYTES_IN_DEVICE_COMMAND, write_ptr);
-    this->manager.issue_queue_push_back(DeviceCommand::NUM_BYTES_IN_DEVICE_COMMAND, LAZY_COMMAND_QUEUE_MODE, this->command_queue_channel);
+    this->manager.issue_queue_push_back(DeviceCommand::NUM_BYTES_IN_DEVICE_COMMAND, LAZY_COMMAND_QUEUE_MODE, this->id);
 }
 
 EnqueueCommandType EnqueueReadBufferCommand::type() { return this->type_; }
 
 // EnqueueWriteBufferCommand section
 EnqueueWriteBufferCommand::EnqueueWriteBufferCommand(
-    uint32_t command_queue_channel,
+    uint32_t id,
     Device* device,
     Buffer& buffer,
     const void* src,
     SystemMemoryManager& manager,
     uint32_t dst_page_index,
     std::optional<uint32_t> pages_to_write) :
-    command_queue_channel(command_queue_channel), manager(manager), src(src), buffer(buffer), dst_page_index(dst_page_index), pages_to_write(pages_to_write.has_value() ? pages_to_write.value() : buffer.num_pages()) {
+    id(id), manager(manager), src(src), buffer(buffer), dst_page_index(dst_page_index), pages_to_write(pages_to_write.has_value() ? pages_to_write.value() : buffer.num_pages()) {
```

Comment:
I would say it makes sense for 'id' attribute for CommandQueue, but this 'id' is confusing cause people may think it's the command id. I would make it 'command_queue_id' here.

Reply:
okay I can name it as `id` on the command queue and in the command classes I will make it `command_queue_id`